css/build.css

```diff
@@ -120,190 +121,115 @@
 }
 /* METRO SIZES */
 .panels.size11 {
-  width: 145px;
+  width: 50%;
   height: 145px;
 }
 
 .panels.size21 {
-  width: 300px;
+  width: 100%;
   height: 145px;
 }
 
 .panels.size22 {
-  width: 300px;
+  width: 100%;
   height: 300px;
 }
 @media (min-width: 420px) {
   .panels.size11 {
-    width: 160px;
+    width: 29%;
     height: 160px;
   }
 
   .panels.size21 {
-    width: 330px;
+    width: 50%;
     height: 160px;
   }
 
   .panels.size22 {
-    width: 330px;
+    width: 50%;
     height: 330px;
   }
 }
+@media (min-width: 600px) {
+  .panels.size11 {
+    width: 30%;
+    height: 185px;
+  }
+
+  .panels.size21 {
+    width: 100%;
+    height: 185px;
+  }
+
+  .panels.size22 {
+    width: 100%;
+    height: 380px;
+  }
+}
 @media (min-width: 640px) {
   .panels.size11 {
-    width: 185px;
+    width: 30%;
     height: 185px;
   }
 
   .panels.size21 {
-    width: 380px;
+    width: 50%;
     height: 185px;
   }
 
   .panels.size22 {
-    width: 380px;
+    width: 50%;
     height: 380px;
   }
 }
 @media (min-width: 1024px) {
   .panels.size11 {
-    width: 230px;
+    width: 30%;
     height: 230px;
   }
 
   .panels.size21 {
-    width: 470px;
+    width: 50%;
     height: 230px;
   }
 
   .panels.size22 {
-    width: 470px;
+    width: 50%;
     height: 470px;
   }
 }
 @media (min-width: 1200px) {
   .panels.size11 {
-    width: 310px;
+    width: 30%;
     height: 310px;
   }
 
   .panels.size21 {
-    width: 630px;
+    width: 50%;
     height: 310px;
   }
 
   .panels.size22 {
-    width: 630px;
+    width: 50%;
     height: 630px;
   }
 }
 @media (min-width: 1300px) {
   .panels.size11 {
-    width: 330px;
+    width: 29%;
     height: 330px;
   }
 
   .panels.size21 {
-    width: 670px;
+    width: 49%;
     height: 330px;
   }
 
   .panels.size22 {
-    width: 670px;
+    width: 49%;
     height: 670px;
   }
 }
-
-[data-container-size="initial"] .panels.size11 {
```

Comment:
Why can we safely remove this?

They were added in [this PR](https://github.com/Fliplet/fliplet-widget-metro/pull/25) to fix [this issue](https://github.com/Fliplet/fliplet-studio/issues/6132).

Reply:
@tonytlwu Yes, you were right we shouldn't remove it.
I've returned that code with percentages instead of the fixed-width so it would also fix our current bug.
[Screencast](https://share.getcloudapp.com/NQu1wZwo)